lightningd/jsonrpc.c

```diff
@@ -150,6 +150,8 @@ static void json_getinfo(struct command *cmd,
 
 	json_object_start(response, NULL);
 	json_add_pubkey(response, "id", &cmd->ld->id);
+	json_add_string(response, "alias", (const char *)cmd->ld->alias);
+	json_add_hex(response, "color", (const void *)cmd->ld->rgb, 3);
```

Comment:
The cast is not needed here, and the length should use `tal_len`. Notice also that both of these fields may be NULL (though they commonly aren't).

Reply:
Yes, update.